sao/mappings/mapping.py

```diff
@@ -0,0 +1,179 @@
+from abc import ABC, abstractmethod
+import numpy as np
+from sao.move_limits.bounds import Bounds
+from sao.util.tools import parse_to_list
+
+
+class Mapping(ABC):
+
+    @property
+    def name(self):
+        return self.__class__.name
+
+    @abstractmethod
+    def g(self, x):
+        """Approximate response function."""
```

Comment:
Not sure if this has to be approximate? The mapping could be exact I suppose? 
```suggestion
        """Evaluate response function."""
```